src/remote_adapter/remote_adapter_client/qt_impl/sdl_remote_adapter_qt_client.cc

```diff
@@ -0,0 +1,125 @@
+#include <iostream>
+
+#include "qt_impl/sdl_remote_adapter_qt_client.h"
+#include "qt_impl/sdl_remote_adapter_receive_thread.h"
+
+namespace lua_lib {
+
+SDLRemoteTestAdapterQtClient::SDLRemoteTestAdapterQtClient(QObject* parent)
+        : QObject(parent) { }
+
+SDLRemoteTestAdapterQtClient::~SDLRemoteTestAdapterQtClient() {
+    if (nullptr != listener_ptr_) {
+        delete(listener_ptr_);
+    }
+
+    if (nullptr != remote_adapter_client_ptr_) {
+        remote_adapter_client_ptr_->clear();
+        delete(remote_adapter_client_ptr_);
+    }
+}
+
+// void SDLRemoteTestAdapterQtClient::connect(const std::string& host, uint32_t port, const std::string& name) {
+//     name_ = name;
+//     try {
+//         remote_adapter_client_ptr_ = new SDLRemoteTestAdapterClient(host, port);
+//         if (0 == open(name)) {
+//             emit connected();
+//             // ToDo (aderiabin): Add incoming message listening
+//         } else {
+//             emit disconnected();
+//         }
+//     } catch (std::exception& e) {
+//         remote_adapter_client_ptr_ = nullptr;
+//         std::cout << e.what() << std::endl;
+//         emit disconnected();
+//     }
+// }
+
+void SDLRemoteTestAdapterQtClient::connectMq(const std::string& host,
+                                             uint32_t port,
+                                             MqParams& in_params,
+                                             MqParams& out_params) {
+    try {
+        remote_adapter_client_ptr_ = new SDLRemoteTestAdapterClient(host, port);
```

Comment:
@aderiabin  Why do you not clean memory?

Reply:
@BSolonenko Fixed in 716e1c1.
Additionaly all connection data is stored in SDLRemoteTestAdapterQtClient now.